www/templates/feats.html

```diff
@@ -0,0 +1,19 @@
+<ion-view view-title="Browse">
```

Comment:
You might want to make the title here to something Feat realated? Unless Browse was chosen on purpose, and we will talk about styling in class tomorrow, but overall you did a great job!